### ISSUE TITLE
Get the last added element

### ISSUE THREAD

**Issue by mitchellholzmann** [legshooter/addel#2]
Something I'm missing in your plugin is that you can get the element that was just added in the events. This is needed so that form attributes can be manipulated.

Or do you have any other way to do this?

Edit: Looks like this happens because you clone the previous target.

**Comment by kobykorman** [comment]
It's the fact I'm including the same object (`target`) on `addel:added`, as I am on `addel:add`.

If you were to have access to the recently added target on `addel:added`, would that be a satisfactory solution for your use case?

**Comment by mitchellholzmann** [comment]
Yea, that would be great and I think it would be better practice if you get the actual target and don't target the original instead of the new cloned element.

**Comment by mitchellholzmann** [comment]
I added an empty clone variable and then changed the clone code to this.

`var clone = target.clone();
clone.insertAfter(target).hide().fadeIn(animation).find(formElements).val(null);`

Then gave the trigger an extra variable to return.
`container.trigger($.Event('addel:added', {target: target, clone: clone}));`

**Comment by kobykorman** [comment]
I think it's better to:

``` javascript
var added = container.find(targetClass).last();
added.find(':input:enabled:visible:first').focus();
```

Because this takes into account the "no visible targets" commented scenario, in which cloning takes no part.

Then, I'll expose the new `added` object, so you could implement like so:

``` javascript
$('.addel').addel().on('addel:added', function (event) {
    event.added.addClass('danger');
});
```

And you'll still have the old target:

``` javascript
$('.addel').addel().on('addel:added', function (event) {
    event.target.removeClass('danger');
    event.added.addClass('danger');
});
```

Cool?

**Comment by mitchellholzmann** [comment]
That looks great, added seems like a better naming convention.

**Comment by kobykorman** [comment]
Released [v1.1.0](https://github.com/legshooter/addel/releases/tag/v1.1.0)

**Comment by mitchellholzmann** [comment]
Just tested it, newest release works great. Thanks for the update.

Closing this now.
